editor.planx.uk/src/ui/shared/DateInput.tsx

```diff
@@ -95,12 +101,15 @@ export default function DateInput(props: Props): FCReturn {
           />
         </Box>
         <Box sx={{ width: INPUT_YEAR_WIDTH }}>
-          <Label variant="body1" htmlFor={`${props.id}-year`} component={"label"}>
+          <Label
+            variant="body1"
+            htmlFor={`${props.id}-year`}
+            component={"label"}
+          >
             Year
           </Label>
           <Input
             value={year || ""}
-            placeholder="YYYY"
```

Comment:
Very happy to remove placeholders here, but should we also flag to content team that any DateInput components should now have descriptions like "For example, 23 4 2005" ? 

Noticed all the GDS examples have this and I wonder if it's how to get a better "usability" score in the absense of official placeholders? https://design-system.service.gov.uk/components/date-input/